abi/src/test/java/org/web3j/abi/FunctionReturnDecoderTest.java

```diff
@@ -20,19 +20,8 @@
 
 import org.junit.jupiter.api.Test;
 
-import org.web3j.abi.datatypes.DynamicArray;
-import org.web3j.abi.datatypes.DynamicBytes;
-import org.web3j.abi.datatypes.Function;
-import org.web3j.abi.datatypes.StaticArray;
-import org.web3j.abi.datatypes.Type;
-import org.web3j.abi.datatypes.Uint;
-import org.web3j.abi.datatypes.Utf8String;
-import org.web3j.abi.datatypes.generated.Bytes16;
-import org.web3j.abi.datatypes.generated.Bytes32;
-import org.web3j.abi.datatypes.generated.StaticArray2;
-import org.web3j.abi.datatypes.generated.StaticArray3;
-import org.web3j.abi.datatypes.generated.StaticArray4;
-import org.web3j.abi.datatypes.generated.Uint256;
+import org.web3j.abi.datatypes.*;
+import org.web3j.abi.datatypes.generated.*;
```

Comment:
please remove the wildcard imports to avoid importing of anything that might break the things, even if is a test class

Reply:
Done